kubernetes/adapter.go

```diff
@@ -92,6 +103,7 @@ func newIngressFromKube(kubeIngress *ingress) *Ingress {
 		namespace:      kubeIngress.Metadata.Namespace,
 		name:           kubeIngress.Metadata.Name,
 		hostName:       host,
+		scheme:         kubeIngress.getAnnotationsString(ingressSchemeAnnotation, "internet-facing"),
```

Comment:
please use the AWS sdk constant here. We probably also should check if the specified value is valid and have a default if not.